packages/backend/src/tokens/request.ts

```diff
@@ -512,6 +510,9 @@ ${error.getFullMessage()}`,
       return handleMaybeHandshakeStatus(authenticateContext, AuthErrorReason.DevBrowserSync, '');
     }
 
+    const isRequestEligibleForMultiDomainSync =
+      authenticateContext.isSatellite && authenticateContext.secFetchDest === 'document';
+
```

Comment:
This is the actual change, correct ?

Reply:
Correct. I will check the clerk_go side shortly but we can push these changes independently

Reply:
The clerk-js change is also important as we couldn't remove the param before for nextjs apps.